x/ibc-rate-limit/types/params.go

```diff
@@ -0,0 +1,65 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store keys.
+var (
+	KeyContractAddress = []byte("contract")
+)
+
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+func NewParams(contractAddress string) (Params, error) {
+	return Params{
+		ContractAddress: contractAddress,
+	}, nil
+}
+
+// default gamm module parameters.
+func DefaultParams() Params {
+	return Params{
+		ContractAddress: "",
+	}
+}
+
+// validate params.
+func (p Params) Validate() error {
```

Comment:
Added a simple test, though I'm having the same issue of the prefix being cosmos instead of osmo. I've spent a while debugging this and can't find the cause, but I'm not too worried about it